deploy/kicbase/Dockerfile

```diff
@@ -18,8 +18,8 @@
 # https://systemd.io/CONTAINER_INTERFACE/
 
 
-# this ARG needs to be global to use it in `FROM` & is updated for new versions of ubuntu:jammy-*
-ARG UBUNTU_JAMMY_IMAGE="ubuntu:jammy-20250819"
+# this ARG needs to be global to use it in `FROM`
+ARG DEBIAN_BOOKWORM_IMAGE="debian:bookworm-slim"
```

Comment:
Why not rename this to BASE_IMAGE? it will make it easier to update or change the image later.